### PR TITLE
added error message for missing lua path and updated readme

### PR DESCRIPTION
**Pull request by arive298**
Created lua formatter extension for vscode. The only code I wrote is on the extension.js , package.json and readme files. The other files were autogenerated by vscode extension generator

### CODE REVIEW

**Comment by ndoma** [comment]
Is `extension.js` able to be tested at all? There is a test file `extension.test.js`, but no tests in it. Just curious.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ Open the command pallet and type `>Lua Format` to run the formatter.
 
 ## Requirements
 
-Formatter only works with lua version 5.1.
+Formatter only works with Lua version 5.1. In addition the complete path to Lua must be specified in the configs such as:
+```
+vscode-metalua-formatter.luaPath = '/usr/local/bin/lua'
```
Comment (cbeeler) [inline README.md]: Everyone will be getting their Lua 5.1 through `lenv`, so a more useful path would be `/home/<user>/.lenv/lua/5.1.5/bin/lua` for Linux and `/Users/<user>/.lenv/lua/5.1.5/bin/lua` for macOS.

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
 		"format",
 		"metalua"
 	],
-	"publisher": "Alex",
+	"publisher": "arivero",
```
Comment (mssonger) [inline package.json]: Not that we don't love our users and want them to have the best experience ever .... but with your info here, it's possible that you could be contacted forever and ever for feature updates and requests and problems and etc etc etc ... Just a thought ... ;->

Also, would you please also set tabs to just two spaces for JSON files (as well as JS files). :-)

Comment (arive298) [reply package.json]: Yeah I know that could cause some work later on but the problem is that I need to make myself a publisher in order to publish the extension into the marketplace.

Comment (mssonger) [reply package.json]: Ah, the marketplace. Cool!!!

**Comment by ryanplusplus** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -8,7 +8,17 @@ Open the command pallet and type `>Lua Format` to run the formatter.
...
+Formatter only works with Lua version 5.1. In addition, the complete path to Lua must be specified in the configs such as:
+```
+Mac
+vscode-metalua-formatter.luaPath = '/Users/<user>/.lenv/lua/5.1.5/bin/lua'
+
+Linux
+vscode-metalua-formatter.luaPath = '/home/<user>/.lenv/lua/5.1.5/bin/lua'
+```
+You can find the complete path by running the following command:
+```which lua```
```
Comment (cbeeler) [inline README.md]: Well maybe. Depends on which lua you are using. If you are using 5.3.5 like most people the number will be wrong.

Comment (arive298) [reply README.md]: I specified that lua 5.1 is required for the formatter to work

**Comment by arive298** [comment]
Well that is the shortcut for linux. For mac the default is shift+alt+f. So I overloaded that shortcut to load the vscode-metalua-formatter only when working on lua files. So for all other files, that same shortcut targets the normal vscode formatter or whatever other formatter you are using.

File: README.md (hunk 3 above)
Comment (cbeeler) [reply README.md]: Yeah, but you are saying you can get the path for it using `which lua`, which may not be true depending how `lenv` was last configured. You may want to expand it to:

```
lenv install 5.1.5
lenv use 5.1.5
which lua
```

or just remove the line all together.

File: README.md [hunk 4]
```diff
@@ -4,11 +4,22 @@ Metalua formatter for vscode. Based on [Format Lua](https://github.com/denglf/Fo
...
+```
+Mac
+vscode-metalua-formatter.luaPath = '/Users/<user>/.lenv/lua/5.1.5/bin/lua'
+
+Linux
+vscode-metalua-formatter.luaPath = '/home/<user>/.lenv/lua/5.1.5/bin/lua'
+```
+You can find the complete path by running the following command:
+```which lua``` 
+For lenv
```
Comment (ryanplusplus) [inline README.md]: Is this a mistake? It looks like you may have started typing something and been interrupted.

Comment (arive298) [reply README.md]: oops

**Pull request merged**